plugins/github/api.ml

```diff
@@ -628,7 +672,7 @@ module Anonymous = struct
       Lwt.try_bind
         (fun () -> query_head repo gref)
         (fun hash ->
-           let id = { Commit_id.owner_name; hash; id = gref } in
+           let id = { Commit_id.owner_name; hash; id = gref; committed_date = "" } in
```

Comment:
Unfinished change here?

Reply:
It was left intentionally like that... It seems it's converting to the `Git` version of a commit id https://github.com/ocurrent/ocurrent/pull/231/files#diff-7928eb03e6563924a0c83bf56953954fa74e5689d4dcf80c02c03cdb4961b9b3R130 

So I thought it could just be left blank given it's about to be dropped by that function?